manifests/bundle.yaml

```diff
@@ -85,9 +102,10 @@ spec:
           mountPropagation: Bidirectional
           name: kubelet-dir
           readOnly: true
-        - mountPath: /etc/storageos/secrets/api
-          name: api-secret
+        - mountPath: /etc/storageos/metrics-exporter-config.yaml
+          name: ondat-metrics-exporter
```

Comment:
Not sure about the naming here. I would prefer `/etc/ondat` over `/etc/storageos`.
But it may be inconsistent with other deployments.
WDYT?

Reply:
changed to `/etc/storageos` afterall to make it consistent with the rest